pwndbg/lib/tempfile.py

```diff
@@ -22,12 +23,15 @@ def tempdir() -> str:
 def cachedir(namespace: str | None = None) -> str:
     """
     Returns and potentially creates a persistent safe cachedir location
-    based on XDG_CACHE_HOME or ~/.cache
+    based on XDG_CACHE_HOME or ~/.cache or LOCALAPPDATA (Windows)
 
     Optionally creates a sub namespace inside the pwndbg cache folder.
     """
-    cachehome = os.getenv("XDG_CACHE_HOME") or os.path.join(os.getenv("HOME", ""), ".cache")
-    cachedir = os.path.join(cachehome, "pwndbg")
+    if sys.platform == "win32":
+        cachehome = os.getenv("LOCALAPPDATA")
+    else:
+        cachehome = os.getenv("XDG_CACHE_HOME") or os.path.join(os.getenv("HOME", ""), ".cache")
```

Comment:
ech, 
`cachehome = os.path.join(os.getenv("HOME", ""), ".cache")`
will return `/.cache/`? but when no envs provided it should use `tempfile.gettempdir()` 🤔